server/src/routers/MenuItemsRouter/parseNutritionInfo.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Parse the nutrition info into a convenient format, 
+ * as in the documentation
+ *
+ * @param {dict} data
+ *
+ * returns {dict} of nutrition data as a json object, 
+ * or null if data is empty
+ */
+
+export default function parseNutritionInfo(data){
+	if(!data || data.length == 0) 
+		return null;
+	return {
+		servingSize: data[2],
```

Comment:
Because the API theoretically change, might it make sense to do: `data[columns.indexOf(columnName)]`? Just a thought, since that's how I've been doing it, but I think hardcoding it is faster and probably fine (since the API is unlikely to change before it's rewritten).